force-app/main/default/classes/SOQL.cls

```diff
@@ -117,7 +137,8 @@ public with sharing class SOQL {
      * @return A list of Case records.
      */
     public static List<Case> getFilterCaseOrigins() {
-        return null; // Replace null with the variable you used to store the result
+        List<Case> myCases = [SELECT Id, Origin, Status FROM Case WHERE Origin NOT IN ('Web', 'Email', '')];
```

Comment:
There are only a few inputs here, so it is easy enough to write them out. I also like to create a Set or List and use that to bind into the query.